packages/cfd/src/Containers/cfd-password-modal.tsx

```diff
@@ -453,28 +474,37 @@ const CFDPasswordForm = (props: TCFDPasswordFormProps) => {
         );
     }
 
-    if (props.should_set_trading_password) {
+    if (should_set_trading_password) {
         return (
             <CFDCreatePasswordForm
-                platform={props.platform}
-                context={props.context}
-                error_message={props.error_message}
-                validatePassword={props.validatePassword}
-                submitPassword={props.submitPassword}
-                has_mt5_account={props.has_mt5_account}
-                is_real_financial_stp={props.is_real_financial_stp}
+                platform={platform}
+                context={context}
+                error_message={error_message}
+                validatePassword={validatePassword}
+                submitPassword={submitPassword}
+                has_mt5_account={has_mt5_account}
+                is_real_financial_stp={is_real_financial_stp}
             />
         );
     }
 
+    const showJuristiction = () => {
+        if (!show_eu_related_content) {
+            return getFormattedJurisdictionCode(jurisdiction_selected_shortcode);
+        } else if (platform === CFD_PLATFORMS.DXTRADE) {
```

Comment:
this condition should be checked before `!show_eu_related_content` since if `!show_eu_related_content` be true, we show juristiction for DerivX which is not correct.